pkg/webui/components/table/cell/cell.styl

```diff
@@ -14,7 +14,7 @@
 
 .cell
   display: table-cell
-  padding: 0 $cs.s
+  padding: 0 0 0 $cs.xs
```

Comment:
That's not a good solution. There should still be padding to both sides, just a bit less.

```css
.cell
  …
  padding: 0 calc($cs.s / 2)
  &:first-child
    padding: 0 calc($cs.s / 2) 0 $cs.s
  &:last-child
    padding: 0 $cs.s 0 calc($cs.s / 2)